src/composites/FeaturesContainer/FilterButton.tsx

```diff
@@ -19,35 +20,54 @@ export const FilterButton = () => {
     setCheckboxStates({ ...checkboxStates, [label]: e.target.checked });
   };
 
-  console.log("Current filter checkboxes status", checkboxStates);
+  // Function to find min and max values in the dataset
```

Comment:
can we move this outside the component, seems like it doesn't need anything from surrounding context.

Reply:
moved this out of FilterButton.